src/advanced/merkelized-list.md

```diff
@@ -145,7 +147,7 @@ T(height > 1, index) = hash(T(height - 1, index * 2)).
 `Proofnode` is a recursively defined structure that's designed to provide
 evidence to client that a certain set of values is contained in a contiguous
 range of indices. One could use several `Proofnode`s to get proof for
-discontiguous set of indexes.
+non-contiguous set of indexes.
```

Comment:
"evidence to client that a certain set of values is contained in a contiguous range of indices."
Personally, I would use "indices" as a plural form for "index", however, I see that in our documentation "indexes" is an absolutely common form, so for the sake of unity of terminology, let's make it "indexes" here and elsewhere. I admit it was my mistake once to try to substitute one for another.
So, please paraphrase as follows: "evidence to the client that a certain set of values is contained in a contiguous range of indexes."
"to get proof for non-contiguous set of indexes." --> "to get a proof for a non-contiguous set of indexes."